DatadogRUM/Sources/RUMVitals/VitalRefreshRateReader.swift

```diff
@@ -172,7 +172,9 @@ extension FrameInfoProvider {
 extension CADisplayLink: FrameInfoProvider {
     var maximumDeviceFramesPerSecond: Int {
         #if swift(>=5.9) && os(visionOS)
-        120
+        // Hardcoded as for now there's no good way of extracting maximum FPS on VisionOS
+        // https://developer.apple.com/documentation/visionos/analyzing-the-performance-of-your-visionos-app/
+        90
```

Comment:
/question I'm good with the comment but why changing the value here?

Reply:
I found it in the doc, maybe the link should https://developer.apple.com/documentation/visionos/analyzing-the-performance-of-your-visionos-app#Inspect-frame-rendering-performance instead?